src/frontend/src/styles/main.css

```diff
@@ -3149,6 +3149,7 @@ input[type="checkbox"] {
   animation-duration: 5s;
   animation-iteration-count: infinite;
   animation-timing-function: ease-in-out;
+  transform: translate(-50%, -50%);
```

Comment:
Why is this necessary?

Reply:
When I disable the animation in the showcase (for screenshots), the logo isn't centered anymore because `translate(-50%, -50%)` was part of the animation. With this change the logo is always centered even if the animation is disabled..

Reply:
noted, thanks!